doc/md/fundamentals/1-basic-syntax/9-operators.md

```diff
@@ -162,7 +162,7 @@ let result = 5 |> double(_) |> increment(_); // (5 * 2) + 1 = 11
 
 Option blocks use the syntax `do ? <block>` to handle optional values of type `?T` without needing nested switch statements. It produces a value of type `?T`, when `<block>` has type `T` and, importantly, introduces the possibility of a break from `<block>.`
```

Comment:
I'm not sure option blocks should be here at all. `do ?` isn't an operator, and `!` is a special post-fix operator that can only be used in a `do ?` block. Maybe just move this to a separate section, perhaps under 10-options-results.

Reply:
You could just mention that `!` is a special operator and point to a later section describing do blocks.